lib/cib/cib_client.c

```diff
@@ -647,7 +647,7 @@ cib_new_variant(void)
     new_cib->cmds->set_slave_all = set_all_secondary; // Deprecated method
 
     new_cib->cmds->upgrade = cib_client_upgrade;
-    new_cib->cmds->bump_epoch = cib_client_bump_epoch;
+    new_cib->cmds->bump_epoch = cib_client_bump_epoch; // Deprecated method
```

Comment:
I'm leaning to not deprecating this one, it's useful, and unused internally only because cibadmin directly calls cib_internal_op() (which external callers can't do)

Reply:
I feel the same way. I made commits for this and `erase` for completeness, to discuss them.